### PR TITLE
Homework8: Marie Ambrazhei

### PR DESCRIPTION
**Pull request by MarieAmbrazhei**

### CODE REVIEW

File: homework8.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    """
+    Play game "Bulls and Cows"
+    """
+
+    digits = list(string.digits)
+    random.shuffle(digits)
+    random_number = ''.join(digits[:4])
+    attempt = 0
+    while True:
+        hidden_four_digit_num = input("ENTER A FOUR-DIGIT NUMBER: ")
```
Comment (Stml89) [inline homework8.py:32]: user can enter non-unique number

File: homework8.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                           statue not in statues_list]
+
+        return len(missing_statues)
+
+    print('Please provide list data type!')
+    return False
+
+
+statues = [6, 2, 3, 8]
+print(count_missed_statues(statues))
```
Comment (Stml89) [inline homework8.py:105]: use assert better then print() here to check that everything works as expected

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework8.py (hunk 1 above)
Comment (MarieAmbrazhei) [reply homework8.py:32]: added validation of unique digits as well as attempt count = 5

File: homework8.py (hunk 2 above)
Comment (MarieAmbrazhei) [reply homework8.py:105]: fixed

File: homework8.py (hunk 1 above)
Comment (Stml89) [reply homework8.py:34]: the user can guess the number as much as he can

**Review by Stml89 — APPROVED** [review]

File: homework8.py (hunk 1 above)
Comment (MarieAmbrazhei) [reply homework8.py:34]: The idea is to give the user only 5 attempts to enter a non-unique  number and an unlimited number of attempts to enter a valid number.

**Pull request merged**